stablehlo/tools/StablehloOptMain.cpp

```diff
@@ -31,9 +31,8 @@ int main(int argc, char **argv) {
   mlir::hlo::registerAllTestPasses();
   mlir::stablehlo::registerPassPipelines();
   mlir::stablehlo::registerPasses();
-  mlir::stablehlo::registerStablehloLegalizeToLinalgPass();
-  mlir::tosa::registerStablehloLegalizeToTosaPassPass();
-  mlir::tosa::registerStablehloPrepareForTosaPassPass();
```

Comment:
What happened to this one? Is it bundled in the above register method?

Reply:
Eh. This is annoying. I thought clicking on "Resolve conversation" with a comment pending would send the comment.

The answer to your question is yes. Here's the relevant generated code (for TOSA, but linalg is similar):

```
inline void registerStablehloTOSATransformsPasses() {
  registerStablehloLegalizeToTosaPass();
  registerStablehloPrepareForTosaPass();
}
```